developing.md

```diff
@@ -43,6 +43,10 @@ To use the plugin locally, run:
 
 This boots a Jenkins master, package the plugin and install it. To run the Jenkins master open http://localhost:8080.
 
+## Running integration tests locally
+
+Please, refer to the [integration tests README](https://github.com/jenkinsci/macstadium-orka-plugin/tree/master/integration-tests).
```

Comment:
I suggest replacing the link with `(integration-tests/README.md)`
This way you will always navigate to the README file on your system and in your branch